modules/apps/27-interchain-accounts/host/types/msgs.go

```diff
@@ -9,6 +9,14 @@ import (
 
 var _ sdk.Msg = (*MsgUpdateParams)(nil)
 
+// NewMsgUpdateParams creates a new MsgUpdateParams instance
```

Comment:
i noticed there was another two instances where this util function could be used, in `transfer/types/msgs_test.go` and `interchainaccounts/controller/types/msgs_test.go`, but could be a followup and not blocker for this pr